x/ibc/07-tendermint/types/msgs.go

```diff
@@ -56,11 +58,8 @@ func (msg MsgCreateClient) Type() string {
 
 // ValidateBasic implements sdk.Msg
 func (msg MsgCreateClient) ValidateBasic() error {
-	if clientType := exported.ClientTypeFromString(msg.ClientType); clientType == 0 {
-		return sdkerrors.Wrap(ErrInvalidClientType, msg.ClientType)
-	}
-	if msg.ConsensusState == nil {
-		return ErrInvalidConsensus
+	if msg.ChainID == "" {
```

Comment:
add "strings" to imported pkgs too

```suggestion
	if strings.TrimSpace(msg.ChainID) == "" {
```